sui_programmability/adapter/transactional-tests/tests/publish/init_public.move

```diff
@@ -1,6 +1,11 @@
+//# init --addresses Test=0x0
```

Comment:
Does this have to go before the copyright? Seems a bit awkward + might make life difficult for the copyright linter

Reply:
There is a longstanding bug/todo that the file has to begin with a command. I should fix that

Reply:
https://github.com/move-language/move/pull/92